LoRaProtocol.h

```diff
@@ -38,6 +43,12 @@ class LoRaProtocol {
 	char sentPacketId [LoRaPacket::idSize + 1];
 	char lastReply    [LoRaPacket::packetSize];
 
+	char ackQueue[ACK_QUEUE_SIZE][LoRaPacket::idSize + 1];
```

Comment:
I kinda think I would either go with a separate templatized queue implementation where we can encapsulate and separately test its functionality, or just use std:queue. There's a lot that can go wrong and dulplicating the implementation between the two queues isn't ideal